SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -945,7 +945,7 @@ void CFlowOutput::AddHistoryOutputFields_ScalarBGS_RES(const CConfig* config) {
   }
 
   switch (config->GetKind_Trans_Model()) {    
-    case TURB_TRANS_MODEL::LM: 
+    case TURB_TRANS_MODEL::LM:  
```

Comment:
And I recommend setting up your IDE to remove trailing spaces automatically.
```suggestion
    case TURB_TRANS_MODEL::LM:
```